### ISSUE TITLE
Разработка адаптеров

### ISSUE THREAD

**Issue by akipcevich** [CodeInside/siu#22]
При разработке адаптера - потребителя столкнулись со следующей ошибкой при отправке сообщения:
 Сбой взаимодействия:
 java.lang.IllegalArgumentException: WSDL bundle://430.3:1/wsdl/RNDR01.wsdl not use http://smev.gosuslugi.ru/rev111111

В xsd схемах указано использование http://smev.gosuslugi.ru/rev111111, в коде адаптера прописано return Revision.rev111111;
Вот сам адрес описания: http://smev-mvf.test.gosuslugi.ru:7777/gateway/services/SID0003325/wsdl

Просим подсказать, в чем может быть проблема, и как ее можно решить.

**Comment by GnusinPavel** [comment]
Добрый день! А вы все зависимые файлы выкачали и поправили пути? Или скачали только одну wsdl по этому адресу http://smev-mvf.test.gosuslugi.ru:7777/gateway/services/SID0003325/wsdl?

**Comment by akipcevich** [comment]
Да, все сделали. Путем проб и ошибок у нас заработало только когда мы нашли так называемую singleWsdl. А вот в зависимостях он путался
